__tests__/__fixtures__/output/node/request/full.js

```diff
@@ -6,14 +6,13 @@ jar.setCookie(request.cookie('bar=baz'), 'https://httpbin.org/anything');
 
 const options = {
   method: 'POST',
-  url: 'https://httpbin.org/anything',
-  qs: {foo: ['bar', 'baz'], baz: 'abc', key: 'value'},
```

Comment:
Prior to this fix `args` was coming back from HTTPBin as the following:

```js
  "args": {
    "baz": "abc", 
    "foo[0]": "bar", 
    "foo[1]": "baz", 
    "key": "value"
  }, 
```

And now:

```js
  "args": {
    "baz": "abc", 
    "foo": [
      "bar", 
      "baz"
    ], 
    "key": "value"
  }, 
```

Reply:
you might also be able to use `URLSearchParams` here too, I haven't checked. Also, unrelated, but I noticed the other day that request is VERY deprecated: https://github.com/request/request#deprecated

Reply:
Request doesn't support `URLSearchParams` for query params it seems:

```js
const request = require('request');

const jar = request.jar();
jar.setCookie(request.cookie('foo=bar'), 'https://httpbin.org/anything');
jar.setCookie(request.cookie('bar=baz'), 'https://httpbin.org/anything');

const qs = new URLSearchParams();
qs.append('foo', 'bar');
qs.append('foo', 'baz');
qs.append('baz', 'abc');
qs.append('key', 'value');

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  qs,
  headers: {
    accept: 'application/json',
    'content-type': 'application/x-www-form-urlencoded'
  },
  form: {foo: 'bar'},
  jar: jar
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
```

```js
{
  "args": {}, 
  "data": "", 
  "files": {}, 
  "form": {
    "foo": "bar"
  }, 
  "headers": {
    "Accept": "application/json", 
    "Content-Length": "7", 
    "Content-Type": "application/x-www-form-urlencoded", 
    "Cookie": "foo=bar; bar=baz", 
    "Host": "httpbin.org", 
    "X-Amzn-Trace-Id": "Root=1-6140e835-7dd0f312641363ff0e30bc4b"
  }, 
  "json": null, 
  "method": "POST", 
  "origin": "REDACTED", 
  "url": "https://httpbin.org/anything"
}
```